flpproto.sh

```diff
@@ -43,7 +44,8 @@ cmake $SOURCEDIR                                              \
     -DFairRoot_DIR=$FAIRROOT_ROOT                               \
     -DROOTSYS=$ROOTSYS \
     ${Configuration_ROOT:+-DConfiguration_ROOT=$Configuration_ROOT} \
-    ${Monitoring_ROOT:+-DMonitoring_ROOT=$Monitoring_ROOT}
+    ${Monitoring_ROOT:+-DMonitoring_ROOT=$Monitoring_ROOT} \
```

Comment:
Hmm how can this work or have ever worked. We expose uppercase variables, so it has to be:

```
${Configuration_ROOT:+-DConfiguration_ROOT=$CONFIGURATION_ROOT}
${Monitoring_ROOT:+-DMonitoring_ROOT=$MONITORING_ROOT}
```